starter-code/routes/drones.js

```diff
@@ -1,21 +1,48 @@
-const express = require('express');
+const express = require("express");
 
 // require the Drone model here
-
+var Drone = require("../models/drone").Drone;
 const router = express.Router();
 
-
-router.get('/drones', (req, res, next) => {
+router.get("/", (req, res, next) => {
   // Iteration #2
+  Drone.find({}, (err, result) => {
+    if (err) {
+      console.log("Error Monongo: ", err);
+    } else {
+      const data = {
+        drones: result
+      };
+      res.render("drones", data);
+    }
+  });
 });
 
-
-router.get('/drones/new', (req, res, next) => {
+router.get("/new", (req, res, next) => {
   // Iteration #3
+  Drone.find({}, (err, result) => {
```

Comment:
Why are you searching for all the drones when going to the page to make a new drone?